kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/scheduler/KafkaScheduler.java

```diff
@@ -159,19 +158,18 @@ private void startApiServer() {
         resources.add(new TopicController(
                 new CmdExecutor(kafkaSchedulerConfiguration, this),
                 this));
-        resources.add(new RecoveryResource(getRecoveryStatusRef()));
 
         // APIs from dcos-commons:
         resources.add(new ConfigResource<>(
-                getConfigState().getConfigStore(),
-                KafkaSchedulerConfiguration.getFactoryInstance()));
+                getConfigState().getConfigStore()));
         resources.add(new TaskResource(
                 getFrameworkState().getStateStore(),
                 taskKiller,
                 kafkaSchedulerConfiguration.getServiceConfiguration().getName()));
-        resources.add(new PlanResource(getPlanManager()));
+        resources.add(new PlansResource(ImmutableMap.of(
+                "deploy",getPlanManager(),
+                "recover",getRepairManager())));
```

Comment:
- Should probably go with 'recovery' to line up with `DefaultScheduler`
- Missing space after `,` here and at L170